### PR TITLE
Installed some internal generics to fix some compiler warnings

### PR DESCRIPTION
**Pull request by tbw777**
Only private simple non generic collections was refactored to generics if no out of class usage detected.
#5524

### CODE REVIEW

**Comment by tbw777** [comment]
Cant push another PR while this waiting something

**Comment by tbw777** [comment]
inconsistent, corrected

**Comment by vieiro** [comment]
Hi, please focus in https://github.com/apache/netbeans/pull/5598 before posting new PRs. 
Once that one is ready for review you can then reopen this one.
Thanks.

**Comment by tbw777** [comment]
@vieiro The PR is ready to be reopen

**Comment by tbw777** [comment]
up

**Comment by tbw777** [comment]
@mbien
@BradWalker

**Comment by tbw777** [comment]
up

File: ide/editor/demosrc/base/org/netbeans/editor/example/Editor.java [hunk 1]
```diff
@@ -523,14 +523,14 @@ public static void main (String args[]) {
     private int        separatorIndex;
     
     private String[] getOpenedFiles() {
-        ArrayList opened = new ArrayList();
+        ArrayList<String> opened = new ArrayList<>();
```
Comment (BradWalker) [inline ide/editor/demosrc/base/org/netbeans/editor/example/Editor.java:526]: I would change this to a List..

**Comment by BradWalker** [comment]
Overall, I'm always in support of code cleanup.

I would suggest that in the future you try and keep the changes in the same domain. This will make code review easier.

For example..
1 - If you say I'm going to clean up module XYZ, then it's appropriate to make a bunch of different changes in there.
2 - If you say I'm going to clean up the List generic, then keep the changes to cleaning up JUST the List interface.

It becomes harder for a volunteer like me to do a code review when a bunch of unrelated changes are requested. Doesn't mean the changes aren't useful, it's just harder to read/understand.

**Comment by tbw777** [comment]
@BradWalker accepted

File: ide/editor/demosrc/base/org/netbeans/editor/example/Editor.java (hunk 1 above)
Comment (tbw777) [reply ide/editor/demosrc/base/org/netbeans/editor/example/Editor.java]: fexied

**Review by mbien — APPROVED** [review]

**Review by BradWalker — APPROVED** [review]

**Pull request merged**